SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/PrivacyConfig/Features/PrivacyFeature.swift

```diff
@@ -86,6 +86,7 @@ public enum PrivacyFeature: String {
     case openFireWindowByDefault
     case behaviorMetrics
     case dataImport
+    case duckAiDataClearing
```

Comment:
The code uses "AI Chat" to refer to duck.ai, since duck.ai is the marketing name (and it did change in the past), it's better to keep using AI Chat to be consistent in the codebase.

Reply:
Makes sense. `duckAiDataClearing` is the C-S-S feature name (already in C-S-S and privacy config). I updated the local feature flag to match it, but I could keep using AI Chat in the local feature flag and only use Duck.ai to refer to the flag already in the privacy config.